desktop/src/main/java/bisq/desktop/main/content/bisq_easy/trade_wizard/review/TradeWizardReviewModel.java

```diff
@@ -120,4 +122,4 @@ public void reset() {
         showCreateOfferSuccess.set(false);
         showTakeOfferSuccess.set(false);
```

Comment:
marketPrice is not reset in reset method

Reply:
Fixed